src/NATS.Client.Core/NatsConnection.RequestReply.cs

```diff
@@ -111,13 +98,34 @@ string Throw()
 
     private NatsSubOpts SetReplyOptsDefaults(NatsSubOpts? replyOpts)
     {
-        var opts = replyOpts ?? DefaultReplyOpts;
+        var opts = replyOpts ?? new NatsSubOpts { MaxMsgs = 1 };
```

Comment:
why aren't we avoiding an extra object allocation here?

Reply:
I will revert...  I guess I was making the (likely wrong) assumption that the compiler would optimize